docs/site/Inside-Loopback-Application.md

```diff
@@ -0,0 +1,79 @@
+---
+lang: en
+title: 'Inside a Loopback Application'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Inside-Loopback-Application.html
+---
+
+## The Shopping Cart Example
+
+The example application [loopback4-example-shopping](https://github.com/strongloop/loopback4-example-shopping)
+is a nice place for developers to start looking on how to use Loopback 4.
+
+Loopback 4 supports Open API specification to define API endpoints. It also builds on the connector infrastructure
+and adds service and entity composition.
+
+In the following sections we will see how the following usecases in the shopping cart example can be configured as API endpoints. Also the example uses a REDIS datasource to save shopping cart items and a mongoDB datasource to persist user and order data.
```

Comment:
IMO, it's better to have the following flow:

1. Explain the API requirements from the front-end perspective by talking about what information is needed to render the home page for a shopping application.

2. Such information needs to be fetched from backend via APIs. It can be multiple API calls but sometimes one call to get aggregated data.

3. Illustrate the backend reality - the needed information comes from various data sources, including databases and services

4. To expose APIs and implement API business logic that requires integration with existing data and services, LB provides the following facilities to make it easy.
...

Reply:
I have split into multiple sections now, to explain each one of the above points.